shell/platform/darwin/ios/framework/Headers/FlutterBinaryMessenger.h

```diff
@@ -39,24 +49,24 @@ FLUTTER_EXPORT
  no reply.
 
  - Parameters:
+   - channel: The channel name.
    - message: The message.
-   - channelName: The channel name.
  */
-- (void)sendBinaryMessage:(NSData* _Nullable)message
-              channelName:(NSString*)channelName;
+- (void)sendOnChannel:(NSString*)channel
```

Comment:
I considered keeping it that way, and I agree it reads more naturally. But the rest of the API (Dart, Java) has the channel name. Same goes for the sibling iOS method to register a handler:
```
-(void)sendOnChannel:(NSString*)channel message:(NSData*)message;
-(void)setMessageHandlerOnChannel:(NSString*)channel binaryMessageHandler:(FlutterBinaryMessageHandler)handler;
```
Further, while the channel name is probably always a short expression, the message may not be. I didn't like the idea if burying the channel name after a multi-line message expression.